pages/courses/[id].js

```diff
@@ -241,6 +246,21 @@ function Course({ course, currentDate }) {
       })
   }
 
+  useEffect(() => {
+    const autoSubscribe = async () => {
+      if (auto_subscribe === 'true' && user && cohort && !userIsRegisteredInCurrentCohort()) {
```

Comment:
JavaScript convention uses camel case for variables (autoSubscribe in this case)

Name of the function is the same, so needs to find better naming also